### PR TITLE
Add RGW support to test script editor

### PR DESCRIPTION
**Pull request by johnpaliotta**
Complete support has been added for TEST.REQUIREMENT_KEY licenese, including:
- Syntax highlighting
- Auto-complete shows all available requirements
- Hover over shows title and description
- E2E tests

### CODE REVIEW

**Comment by johnpaliotta** [comment]
@aytey I updated the launch.json because the syntax of the command changed.

File: syntax/vcast.tst.grammar.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
       "name": "keyword"
     },
     {
-      "match": "^TEST\\.(?:UNIT|SUBPROGRAM|NAME|SCRIPT_FEATURE|STUB)(:|$)(.*)$",
+      "match": "^TEST\\.(?:UNIT|SUBPROGRAM|NAME|SCRIPT_FEATURE||REQUIREMENT_KEY)(:|$)(.*)$",
```
Comment (aytey) [inline syntax/vcast.tst.grammar.json:18]: Accidental `||` in here?

Comment (aytey) [reply syntax/vcast.tst.grammar.json:18]: Also, where did STUB go?

Comment (johnpaliotta) [reply syntax/vcast.tst.grammar.json:18]: fixed

**Comment by johnpaliotta** [comment]
@Zbigor  @aytey Tests look good.  Should we / can we test the work I do on right click load?  As you know when there is a requirement that was entered via the extension, it gets put in like this:
```
TEST.REQUIREMENT_KEY:FR20 | Clearing a table resets orders for all seats
```
On the load action I turn that into a comment containing the title part and a second line: TEST.REQUIREMENT_KEY:FR20
So you could test this two ways, ensure that the TEST.REQUIREMENT_KEY:FR20 line is there, or ensure that clicast can load the script without any errors.

**Pull request merged**